.github/workflows/tests.yaml

```diff
@@ -127,19 +132,25 @@ jobs:
         run: |
           google-chrome --version
           xvfb-run --server-args="-screen 0, 1280x720x24" --auto-servernum \
-          chromedriver --port=4444 --url-base=/wd/hub &> ./logs/chromedriver.log &
```

Comment:
Currently this path is by default expected [in the WebDriverTestCase](https://github.com/php-webdriver/php-webdriver/blob/main/tests/functional/WebDriverTestCase.php#L24).

This is causing chrome test to fail: https://github.com/php-webdriver/php-webdriver/pull/874/checks?check_run_id=2170530712

So I guess this should be kept.

Reply:
reverted